chromium_src/chrome/browser/ui/views/side_panel/side_panel_util.cc

```diff
@@ -25,10 +29,12 @@ void SidePanelUtil::PopulateGlobalEntries(Browser* browser,
         ->CreateAndRegisterEntry(global_registry);
   }
 
+#if BUILDFLAG(ENABLE_AI_CHAT)
   if (ai_chat::features::IsAIChatEnabled()) {
```

Comment:
If this call is compile time expression, we may ignore the BUILDFLAG check

Reply:
It is runtime